pages/jobs/apiture-api-architect.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'API Architect'
+date: 22/06/2021
+category: job category
+closingOn: 22/08/2021
+---
```

Comment:
@DavidBiesack thanks for the PR!
you need to add `company` object as here https://raw.githubusercontent.com/asyncapi/website/master/pages/jobs/ui-ux-dx-designer.md, so your PR should include the logo too